website/docs/r/en_destination_custom_email.html.markdown

```diff
@@ -17,6 +17,7 @@ resource "ibm_en_destination_custom_email" "custom_domain_en_destination" {
   instance_guid         = ibm_resource_instance.en_terraform_test_resource.guid
   name                  = "Custom Email EN Destination"
   type                  = "smtp_custom"
+  verification_type     = ""
```

Comment:
why we want to empty verification_type?

Reply:
I will correct documentation, it's optional parameter.